widgets/SendGrid-batchsource.json

```diff
@@ -0,0 +1,399 @@
+              {
+                "id": "Senders",
+                "label": "Senders"
+              },
+              {
+                "id": "SingleSends",
+                "label": "Single Sends"
+              }
+            ],
+            "delimiter": ",",
+            "default": ""
+          }
+        },
+        {
+          "name": "dataSourceStats",
+          "label": "Statistic Objects",
+          "widget-type": "multi-select",
+          "widget-attributes": {
+            "options": [
+              {
+                "id": "AdvancedStats",
+                "label": "Advanced Stats"
+              },
+              {
+                "id": "CategoryStats",
+                "label": "Category Stats"
+              },
+              {
+                "id": "GlobalStats",
+                "label": "Advanced Stats"
+              },
+              {
+                "id": "_suppressions",
+                "label": "Suppression Objects"
+              },
+              {
+                "id": "Bounces",
+                "label": "Bounces"
+              },
+              {
+                "id": "GlobalUnsubscribes",
+                "label": "Global Unsubscribes"
+              },
+              {
+                "id": "GroupUnsubscribes",
+                "label": "Group Unsubscribes"
+              }
+            ],
+            "delimiter": ",",
+            "default": ""
+          }
+        },
+        {
+          "name": "dataSourceSuppressions",
+          "label": "Suppression Objects",
+          "widget-type": "multi-select",
+          "widget-attributes": {
+            "options": [
+              {
+                "id": "Bounces",
+                "label": "Bounces"
+              },
+              {
+                "id": "GlobalUnsubscribes",
+                "label": "Global Unsubscribes"
+              },
+              {
+                "id": "GroupUnsubscribes",
+                "label": "Group Unsubscribes"
+              }
+            ],
+            "delimiter": ",",
+            "default": ""
+          }
+        },
+        {
+          "name": "dataSourceFields",
+          "label": "Fields",
+          "widget-type": "multi-select",
+          "widget-attributes": {
+            "options": [
+              {
+                "id" : "address",
+                "label": "address"
+              },
+              {
+                "id" : "address_2",
+                "label": "address_2"
+              },
+              {
+                "id" : "city",
+                "label": "city"
+              },
+              {
+                "id" : "contracts_count",
+                "label": "contracts_count"
+              },
+              {
+                "id" : "country",
+                "label": "country"
+              },
+              {
+                "id" : "created",
+                "label": "created"
+              },
+              {
+                "id" : "created_at",
```

Comment:
please check if we use camel case in others. So it's the same way everywhere.

Reply:
it is the field name to query(as it is provided by SendGrid API), and camel case is not applicable here